### ISSUE TITLE
Tutorial 7 (PyTorch) Graph Neural Networks: Non-linearity in the forward pass for the GCN Layer missing?

### ISSUE THREAD

**Issue by choudharynishu** [phlippe/uvadlc_notebooks#147]
According to the equation given before the implementation of the GCN Layer, the output after normalization should pass through a non-linearity but I don't see any non-linearity function used in the forward hook. Is this a mistake ?
